pkgs/by-name/de/demucs-cpp/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  blas,
+}:
+stdenv.mkDerivation rec {
+  pname = "demucs-cpp";
+  version = "0.0.4-alpha";
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "sevagh";
+    repo = "demucs.cpp";
+    rev = "v${version}";
+    hash = "sha256-STbD2bJnMhcglkNKzExZbteFbmT1lDnfnhJPuV3TgAk=";
+  };
+
+  buildInputs = [ blas ];
+
+  nativeBuildInputs = [ cmake ];
+
+  installPhase = ''
```

Comment:
I'm not sure on this point, but could the `installPhase` be skipped completely if a prefix were passed as a configuration flag?